src/app/content/utils/urlUtils.ts

```diff
@@ -25,19 +25,16 @@ export const getBookPageUrlAndParams = (
     pageUid: stripIdVersion(page.id),
   };
 
-  if (!('version' in params.book) && (!BOOKS[book.id] || book.version !== BOOKS[book.id].defaultVersion)) {
-    const paramsWithVersion = { ...params, book: {...params.book, version: book.version}};
-    return { params: paramsWithVersion, state, url: contentRoute.getUrl(paramsWithVersion) };
-  }
-
   return {params, state, url: contentRoute.getUrl(params)};
 };
 
 export const getUrlParamsForBook = (
   book: Pick<Book, 'id' | 'tree' | 'title' | 'version'> & Partial<{slug: string}>
 ): Params['book'] => {
   if ('slug' in book && book.slug && BOOKS[book.id]) {
-    return {slug: book.slug};
+    return book.version === BOOKS[book.id].defaultVersion
+      ? {slug: book.slug}
+      : {slug: book.slug, version: book.version};
```

Comment:
This is not exactly related to this PR but I've noticed that content links dont get properly versioned, so i included it here

Reply:
I also thought maybe about making a map that `toRelativeUrl` could use because if you navigate to a book with an uuid, and that books has a slug, then the urls will be '../../slug/page/pageThatChanges'. It works but just thought it might be nice.

Reply:
if you navigate to a book with a uuid and it renders content links with the slug that might be a bug, but lets not worry about this for now.